docs/production-deployment/cloud/audit-logging.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-id: audit-logging
-title: Audit Logging - Temporal Cloud feature guide
-sidebar_label: Audit Logging
-description: Audit Logging in Temporal Cloud provides forensic information, integrating with Amazon Kinesis for secure data handling and supporting key Admin and API Key operations. This streamlines audit and compliance processes.
-slug: /cloud/audit-logging
+id: audit-logging-gcp
+title: Audit Logging - GCP Pub/Sub
+sidebar_label: GCP Pub/Sub
+description: Audit Logging in Temporal Cloud provides forensic information, integrating with GCP Pub/Sub for secure data handling and supporting key Admin and API Key operations. This streamlines audit and compliance processes.
+slug: /cloud/audit-logging-gcp
```

Comment:
I think this is the wrong page contents? Seems it's the `audit-logging-gcp.mdx` one ?

Reply:
Ugh. Yup.

Reply:
Fixed! Good catch.